client/js/components/user/CustomerSettings/CustomerSettingsContact.vue

```diff
@@ -0,0 +1,226 @@
+        new: Translator.trans('customer.contact.new'),
+        add: Translator.trans('customer.contact.add'),
+        abort: Translator.trans('abort'),
+        update: Translator.trans('customer.contact.update'),
+        noEntries: Translator.trans('customer.contact.no'),
+        delete: Translator.trans('customer.contact.delete')
+      }
+    }
+  },
+
+  computed: {
+    ...mapState('customerContact', {
+      contacts: 'items'
+    })
+  },
+
+  methods: {
+    ...mapActions('customerContact', {
+      createContact: 'create',
+      fetchContacts: 'list',
+      deleteContact: 'delete',
+      saveContact: 'save'
+    }),
+
+    ...mapMutations('customerContact', {
+      updateContact: 'setItem'
+    }),
+
+    reset () {
+      this.showContactForm = false
+    },
+
+    createOrUpdateContact (id) {
+      if (id === 'new') {
+        const payload = {
+          type: 'CustomerContact',
+          attributes: {
+            title: this.customerContact.title,
+            phoneNumber: this.customerContact.phoneNumber ? this.customerContact.phoneNumber : null,
+            text: this.customerContact.text ? this.customerContact.text : null,
+            visible: this.customerContact.visible,
+            eMailAddress: this.customerContact.eMailAddress ? this.customerContact.eMailAddress : null
+          }
+        }
+        this.createContact(payload).then((response) => {
+          this.getContacts()
+          dplan.notify.notify('confirm', Translator.trans('confirm.saved'))
+        })
+      } else {
+        const payload = {
+          id: id,
+          type: 'CustomerContact',
+          attributes: {
+            title: this.customerContact.title,
+            phoneNumber: this.customerContact.phoneNumber ? this.customerContact.phoneNumber : null,
+            text: this.customerContact.text ? this.customerContact.text : null,
+            visible: this.customerContact.visible,
+            eMailAddress: this.customerContact.eMailAddress ? this.customerContact.eMailAddress : null
+          }
+        }
+        this.updateContact(payload)
+        this.saveContact(id).then(() => {
+          dplan.notify.notify('confirm', Translator.trans('confirm.saved'))
+        })
+      }
+      this.$refs.editableList.toggleFormVisibility(false)
+      this.resetForm()
+    },
+
+    getContacts () {
+      this.fetchContacts({
+        fields: {
+          CustomerContact: [
+            'title',
+            'phoneNumber',
+            'text',
+            'visible',
+            'eMailAddress'
+          ].join()
+        }
+      })
+    },
+
+    resetForm () {
+      this.customerContact.title = ''
+      this.customerContact.phoneNumber = ''
+      this.customerContact.eMailAddress = ''
+      this.customerContact.visible = false
+      this.customerContact.text = ''
+    },
+
+    updateForm (index) {
+      const currentData = this.contacts[index].attributes
+      this.customerContact.title = currentData.title
```

Comment:
This looks like the title is mandantory. Why isn't it then required in DpFaqSupportCard?

Reply:
title is not required in supportcard like discussed in zoom

Reply:
data comming from the backend all have a title afaik. 
At least the Form was designed to have a required ttile.

Reply:
so we leave it like it is here....